app/assets/stylesheets/components/buttons.scss

```diff
@@ -0,0 +1,27 @@
+.button {
+  $button-colors: (
+    "primary": $button-primary,
+    "secondary": $button-seconday,
+  );
+
+  border-radius: 0.2em;
+  box-shadow: 0 0.05em 0.25em 0 rgba(0, 0, 0, 0.5);
+  color: $white !important;
```

Comment:
Bootstrap is overriding some styles are turning the button text to a blue color when in a hover or focus state. I'll define a hover and focus state in the button component to avoid important.